### ISSUE TITLE
[Feature] Version management views should be split into two

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/brainrender-napari#129]
**Is your feature request related to a problem? Please describe.**
There are many atlases and we plan to add more - but we expect users to typically only download and interact with a few of them. So it should be easier to distinguish the local and not-local atlases.

**Describe the solution you'd like**
The version management widget should contain two views, one for the locally available atlases (with a signal to show which ones can be updated) and one for the not-yet-downloaded atlases.

**Describe alternatives you've considered**
Could also colour-code rows, but this is both more difficult to implement and less clear to users, I think.

**Additional context**
May be worth refactoring the code to use proxy models instead of different views in this process?

**Comment by kira1228** [comment]
Hi @alessandrofelder 

Is this issue resolved with [PR 183](https://github.com/brainglobe/brainrender-napari/pull/183) ?

**Comment by alessandrofelder** [comment]
No, I think this would be more involved: here we are envisaging two separate view instances in the widget, one that contains the local atlases, and one that show that remote-only atlases.
